libraries/gpu-gl-common/src/gpu/gl/GLTextureTransfer.cpp

```diff
@@ -218,7 +236,19 @@ void GLTextureTransferEngineDefault::updateMemoryPressure() {
 
     Backend::textureResourceIdealGPUMemSize.set(idealMemoryAllocation);
     size_t unallocated = idealMemoryAllocation - totalVariableMemoryAllocation;
-    float pressure = (float)totalVariableMemoryAllocation / (float)allowedMemoryAllocation;
+    float pressure = 0;
+
+    if (useAvailableGlMemory) {
+        float totalMem = GLBackend::getTotalMemory();
+        float availMem = GLBackend::getAvailableMemory() - AUTO_RESERVE_TEXTURE_MEMORY;
```

Comment:
Why the `- AUTO_RESERVER_TEXUTRE_MEMORY` / why is the 64MB value correct? Perhaps add a comment?

Reply:
In that branch, we're trying to just available memory. The 64M is there because I'm supposing completely filling it might be undesirable (eg, the desktop UI or a web browser might need some).

64M is kind of a randomly picked value. My logic was that on the low end you get 1 GB-ish, and ideally little should be left unused.